pouta_blueprints/tasks/provisioning_tasks.py

```diff
@@ -38,7 +38,7 @@ def get_provisioning_type(token, instance_id):
     return pbclient.get_plugin_data(plugin_id)['name']
 
 
-@celery_app.task(name="pouta_blueprints.tasks.run_update")
+@celery_app.task(name="pouta_blueprints.tasks.run_update", expires=60)
```

Comment:
This was added because if you disable the provisioning worker(s), the update tasks spawned by periodical task worker every minute would keep piling up.